docs/relying-parties/reference/sub-plat-overview.md

```diff
@@ -22,6 +22,16 @@ Importantly, subscriptions are not locked to any one FxA-attached product by def
 
 If a user cancels their subscription, this metadata goes away and your product can go back to treating them as a non-subscribed user.
 
+## Previews
+
+For a preview of the different pages and components that make up the Subscription Platform, as well as emails sent to customers, we have Storybook builds available.
+
+- Pages and components
+  - [Checkout with existing Firefox Account](https://storage.googleapis.com/mozilla-storybooks-fxa/commits/latest/fxa-payments-server/index.html?path=/story/routes-product--subscribing-with-new-account)
+  - [Checkout without Firefox Account](https://storage.googleapis.com/mozilla-storybooks-fxa/commits/latest/fxa-payments-server/index.html?path=/story/routes-checkout--subscribing-with-a-new-account)
+  - [Subscription Management](https://storage.googleapis.com/mozilla-storybooks-fxa/commits/latest/fxa-payments-server/index.html?path=/story/routes-subscriptions--subscribed-with-web-subscription)
+- [Email templates](https://storage.googleapis.com/mozilla-storybooks-fxa/commits/latest/fxa-auth-server/index.html?path=/story/subplat-emails-templates-downloadsubscription--download-subscription)
```

Comment:
These links could unknowingly become out of date if we change the name of any of these stories or components. I wonder if some of them could be linked differently to auto-redirect to where they should be, like at least this one can be: 

https://storage.googleapis.com/mozilla-storybooks-fxa/commits/latest/fxa-auth-server/index.html?path=/story/subplat-emails-templates

Which redirects to the first email under "templates". Not a big deal if the others can't redirect nicely.